magenta/models/shared/melody_rnn_generate.py

```diff
@@ -125,13 +126,21 @@ def run_with_flags(melody_rnn_sequence_generator):
   if not os.path.exists(FLAGS.output_dir):
     os.makedirs(FLAGS.output_dir)
 
+
+#primer_sequence from written melody or given midi file or random note between midi #48 = 4C to midi #84 = 7C
```

Comment:
@danabo doesn't melody_rnn_sequence_generator already do this?

https://github.com/tensorflow/magenta/blob/master/magenta/models/shared/melody_rnn_sequence_generator.py#L122